main/mariadb/APKBUILD

```
@@ -184,7 +183,6 @@ check() {
 	# exclude test-connect which seems to be buggy. testsuite does not set port env var
 	# ma_test_loghandler_readonly seems to fail if /etc/my.conf.d is missing
 	ctest -E '(test-connect|ma_test_loghandler_readonly)'
```

```suggestion
	ctest -E 'test-connect'
```
I also worked on upgrading to 10.4.6 and the test ma_test_loghandler_readonly passes on my machine (x86_64).

---

Thanks, added